tests/test_locales.py

```diff
@@ -211,34 +315,110 @@ def test_format_timeframe(self):
-        assert self.locale._format_timeframe("minutes", -10) == "10 minutos"
-        assert self.locale._format_timeframe("hour", -1) == "una hora"
-        assert self.locale._format_timeframe("hours", -3) == "3 horas"
-        assert self.locale._format_timeframe("hours", -11) == "11 horas"
-        assert self.locale._format_timeframe("day", -1) == "un día"
-        assert self.locale._format_timeframe("days", -2) == "2 días"
-        assert self.locale._format_timeframe("days", -12) == "12 días"
-        assert self.locale._format_timeframe("week", -1) == "una semana"
-        assert self.locale._format_timeframe("weeks", -2) == "2 semanas"
-        assert self.locale._format_timeframe("weeks", -3) == "3 semanas"
-        assert self.locale._format_timeframe("month", -1) == "un mes"
-        assert self.locale._format_timeframe("months", -3) == "3 meses"
-        assert self.locale._format_timeframe("months", -13) == "13 meses"
-        assert self.locale._format_timeframe("year", -1) == "un año"
-        assert self.locale._format_timeframe("years", -4) == "4 años"
-        assert self.locale._format_timeframe("years", -14) == "14 años"
+        assert self.locale._format_timeframe("seconds", -1) == "hace 1 segundos"
+        assert self.locale._format_timeframe("seconds", -9) == "hace 9 segundos"
+        assert self.locale._format_timeframe("seconds", -12) == "hace 12 segundos"
+        assert self.locale._format_timeframe("minute", -1) == "hace un minuto"
+        assert self.locale._format_timeframe("minutes", -2) == "hace 2 minutos"
+        assert self.locale._format_timeframe("minutes", -10) == "hace 10 minutos"
+        assert self.locale._format_timeframe("hour", -1) == "hace una hora"
+        assert self.locale._format_timeframe("hours", -3) == "hace 3 horas"
+        assert self.locale._format_timeframe("hours", -11) == "hace 11 horas"
+        assert self.locale._format_timeframe("day", -1) == "hace un día"
+        assert self.locale._format_timeframe("days", -2) == "hace 2 días"
+        assert self.locale._format_timeframe("days", -12) == "hace 12 días"
+        assert self.locale._format_timeframe("week", -1) == "hace una semana"
+        assert self.locale._format_timeframe("weeks", -2) == "hace 2 semanas"
+        assert self.locale._format_timeframe("weeks", -3) == "hace 3 semanas"
+        assert self.locale._format_timeframe("month", -1) == "hace un mes"
+        assert self.locale._format_timeframe("months", -3) == "hace 3 meses"
+        assert self.locale._format_timeframe("months", -13) == "hace 13 meses"
+        assert self.locale._format_timeframe("year", -1) == "hace un año"
+        assert self.locale._format_timeframe("years", -4) == "hace 4 años"
+        assert self.locale._format_timeframe("years", -14) == "hace 14 años"
 
 
 @pytest.mark.usefixtures("lang_locale")
 class TestFrenchLocale:
     def test_ordinal_number(self):
+
+
+        assert self.locale.ordinal_number(0) == "0ème"
```

Comment:
This is an edge case we're missing within ```_ordinal_number``` method for the FrenchBaseLocale in ```arrow\locales.py```. Feel free to add the appropriate edge case with this method to fix this.